### PR TITLE
[Edgecore][as9716-32d] Config TI retimer for SFP

### PR DESCRIPTION
**Pull request by brandonchuang**
Config TI retimer chip for SFP to make the traffic work.

### CODE REVIEW

File: packages/platforms/accton/x86-64/as9716_32d/platform-config/r0/src/python/x86_64_accton_as9716_32d_r0/__init__.py [hunk 1]
```diff
@@ -34,6 +34,25 @@ def ir3570_check():
         return -1
     return ret
 
+def config_sfp_retimer():
+    cmd_list = [
+        "i2cset -f -y 22 {} 0x7 0x3",   # Set Mux(retimer) to 2x10G XFI
```
Comment (paulmenzel) [inline packages/platforms/accton/x86-64/as9716_32d/platform-config/r0/src/python/x86_64_accton_as9716_32d_r0/__init__.py:39]: Add a space before (?

**Review by paulmenzel — COMMENTED** [review]
Please spell *brandonchuang* as *Brandon Chuang* (also the commit author with `git commit --amend --author="Brandon Chuang <…>"`.

Could you please describe the issue? Why has nobody noticed this until now? Is it hardware dependent?

File: packages/platforms/accton/x86-64/as9716_32d/platform-config/r0/src/python/x86_64_accton_as9716_32d_r0/__init__.py (hunk 1 above)
Comment (brandonchuang) [reply packages/platforms/accton/x86-64/as9716_32d/platform-config/r0/src/python/x86_64_accton_as9716_32d_r0/__init__.py]: Done

**Comment by brandonchuang** [comment]
> Please spell _brandonchuang_ as _Brandon Chuang_ (also the commit author with `git commit --amend --author="Brandon Chuang <…>"`.
> 
> Could you please describe the issue? Why has nobody noticed this until now? Is it hardware dependent?

I think these two ports are rarely used.

**Review by sonoble — APPROVED** [review]
Specific to the AS9716

**Pull request merged**